sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
my [last submitted patch](https://www.drupal.org/node/2643692#comment-10830420) omitted test_dependencies and passed all tests